Classes/EventListener/EventListenerLocator.php

```diff
@@ -55,16 +55,17 @@ public function initializeObject()
      * Returns event listeners for the given event (type)
      *
      * @param EventInterface $event
-     * @return \Generator
+     * @return \callable[]
```

Comment:
I love generators but I don't see an advantage in using them here?
With an array of `\callable` we can later extend this to work with closures etc. Besides we don't move the object instantiation to some `DTO`.
Lastly, this is a preparation for the projection replay I'm currently working on